mkosi/__init__.py

```diff
@@ -5184,10 +5205,15 @@ def create_parser() -> ArgumentParserMkosi:
     group.add_argument("-r", "--release", help="Distribution release to install")
     group.add_argument("-m", "--mirror", help="Distribution mirror to use")
     group.add_argument(
-        "--repositories", action=CommaDelimitedListAction, default=[], help="Repositories to use", metavar="REPOS"
+        '--repositories',
+        metavar='REPOS',
+        action=CommaDelimitedListAction,
+        default=[],
+        help='Repositories to use',
     )
     group.add_argument(
-        "--use-host-repositories",
+        '--use-host-repositories',
```

Comment:
Just as a stand in for the others: Is this really necessary? That's an awful lot of churn.

Reply:
In general, single quotes are the standard for Python. I'm adding new lines with single quotes, and it'd be awkward to have a lines with double quotes and single quotes in a single invocation.

Reply:
I wouldn't go as far as saying they are standard. 
> In Python, single-quoted strings and double-quoted strings are the same. This PEP does not make a recommendation for this. Pick a rule and stick to it. When a string contains single or double quote characters, however, use the other one to avoid backslashes in the string. It improves readability.

With the large number of projects adopting black, I would say that double quotes are becoming ever more prevalent, but I'm not going to start an argument about this, since I know we disagree on this. :) That being said, I agree that having different quotes on lines directly following each other would be weird, but the diffstat would be smaller if they added lines had the same quotes as the ones around them.

Reply:
Yeah, but then I'd have to add lines with double quotes, and I'd prefer not to do that ;)

Reply:
I know. [But look how much cleaner the diffstat could be](https://github.com/behrmann/mkosi/commit/45e561cd6dbaa6466fe5da0c1bf0be8a63d30d3c) ;)

Reply:
Just throwing out my preference for double quotes. On the other hand, I don't care at all as long as we primarily use one over the other.

Reply:
I think the priority of single quotes over double quotes was fairly well established as pythonic. Old manuals generally said "use single unless you have a reason to use double". Python code was written in a minimalistic way, much more than nowadays. Hence it was natural to prefer a character which uses 50% of black.

```pycon
>>> ["abc", "123", "\"", "'"]
['abc', '123', '"', "'"]
```
The interpreter implements the same rule ;)

Nevertheless, I [un]changed the quotes to not add any single-quotes. The diff is much smaller ;)